core/cas-server-core-tickets/src/main/java/org/apereo/cas/ticket/support/RememberMeDelegatingExpirationPolicy.java

```diff
@@ -26,16 +28,23 @@
      * deserialization the field is null.
      */
     private static final Logger LOGGER = LoggerFactory.getLogger(RememberMeDelegatingExpirationPolicy.class);
-    
+
+    @JsonProperty
     private ExpirationPolicy rememberMeExpirationPolicy;
 
+    @JsonProperty
     private ExpirationPolicy sessionExpirationPolicy;
 
     /**
      * Instantiates a new Remember me delegating expiration policy.
      */
     public RememberMeDelegatingExpirationPolicy() {}
 
+    public RememberMeDelegatingExpirationPolicy(ExpirationPolicy rememberMeExpirationPolicy, ExpirationPolicy sessionExpirationPolicy) {
+        this.rememberMeExpirationPolicy = rememberMeExpirationPolicy;
+        this.sessionExpirationPolicy = sessionExpirationPolicy;
+    }
+
     @PostConstruct
```

Comment:
Is there any reason for not to use a constructor with both fields as parameters and do this PostConstruct check on it?
If I'm right, all the field != null checks are not needed since the constraint is applied after construction